test/libsolidity/StandardCompiler.cpp

```diff
@@ -2282,11 +2231,8 @@ BOOST_AUTO_TEST_CASE(ethdebug_ethdebug_output)
 	for (auto const& test: tests)
```

Comment:
```suggestion
	for (auto const& [standardJsonToCompile, optionalCheck]: tests)
```

or something like that would make the tests more easily digestible :)